src/mainwindow.cpp

```diff
@@ -11943,7 +11943,6 @@ Qt::SortOrder MainWindow::toQtOrder(int order) {
 void MainWindow::updatePanelsSortOrder() {
     updateNotesPanelSortOrder();
     reloadNoteSubFolderTree();
-    reloadTagTree();
```

Comment:
And how how will the tag tree be reloaded after updating the panel sort order?

Reply:
tag tree is already reloaded once after panel sort order is updated, isn't it?

Reply:
You mean actually in that method? I don't think so. Is it?

Reply:
In `updateNotesPanelSortOrder()`, `loadNoteDirectoryList()` gets called, which calls `setupTags` and tag tree is reloaded. See Call stack 2 in OP.

Reply:
**1** already does almost everything. **2** seems redundant except for the function `updateNotesPanelSortOrder` which basically updates some checkboxes I think. (Unless I am missing something here)

Reply:
Looks ok. thank you. But can you please instead of removing the method call comment it out and add a comment explaining that you just said just make clear what we need to achieve updatePanelsSortOrder().

Reply:
Done